docs/_integration-with-exisiting-apps-java.md

```diff
@@ -355,28 +332,28 @@ Now your activity is ready to run some JavaScript code.
 
 You have now done all the basic steps to integrate React Native with your current application. Now we will start the [Metro bundler][metro] to build the `index.bundle` package and the server running on localhost to serve it.
 
-##### 1. Run the packager
+#### 1. Run the packager
 
 To run your app, you need to first start the development server. To do this, run the following command in the root directory of your React Native project:
 
 ```shell
 $ yarn start
 ```
 
-##### 2. Run the app
+#### 2. Run the app
 
 Now build and run your Android app as normal.
 
 Once you reach your React-powered activity inside the app, it should load the JavaScript code from the development server and display:
 
 ![Screenshot](/docs/assets/EmbeddedAppAndroid.png)
 
-### Creating a release build in Android Studio
+#### Creating a release build in Android Studio
 
 You can use Android Studio to create your release builds too! It’s as quick as creating release builds of your previously-existing native Android app. There’s one additional step, which you’ll have to do before every release build. You need to execute the following to create a React Native bundle, which will be included with your native Android app:
 
 ```shell
-$ npx react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/com/your-company-name/app-package-name/src/main/assets/index.android.bundle --assets-dest android/com/your-company-name/app-package-name/src/main/res/
+$ yarn react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res/
```

Comment:
Can you explain the reason for this change? Since CLI extraction from core, `npx` is a recommended way to call the RN scripts.

Also it would be nice if you can add the package manager Tabs (example: https://reactnative.dev/docs/next/typescript) to the code blocks which require calling `yarn` (or `npm`) directly.